### PR TITLE
Fix runtime not being python (cont.)

### PR DESCRIPTION
**Pull request by jameskbride**
# Context
This is a continuation of the work done in #745, which should address #741 by enabling non-Python runtimes to be specified in `provider.runtime` attribute of the serverless config. 

# Changes
- Fix runtime not being python
- Added test
- Ignoring test-generated packages.
- Adding .python-version file for the required version of Python.
- Updating the CONTRIBUTING.md instructions.

# Testing
- Unit tested

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -12,9 +12,17 @@ Welcome, and thanks in advance for your help!
 
 ## Setup
 
-It is recommended to use Node v14 or v16 for development.
-
-Then, to begin development fork repository and run `npm install` in its root folder.
+Pre-Reqs:
+* Python 3.7
+* [poetry](https://python-poetry.org/docs/) (if you use multiple versions of Python be sure to install it with python 3.7)
+* Perl (used in the tests)
```
Comment (pgrzesik) [inline CONTRIBUTING.md:18]: I don't think Perl is really needed

**Review by pgrzesik — COMMENTED** [review]
Overall looks nice, I have one comment, please let me know what do you think

File: CONTRIBUTING.md (hunk 1 above)
Comment (jameskbride) [reply CONTRIBUTING.md:18]: Apologies, just now seeing this. I can remove the perl reference if you want, but it is [required](https://github.com/mklenbw/serverless-python-requirements/blob/7715237f8ac41b5889ea2388affcc1c7f7493d2d/test.js#L53) if we want to run the tests. I actually couldn't get the tests to run without it.

Comment (pgrzesik) [reply CONTRIBUTING.md:18]: oh, that's interesting to learn. Thank you @jameskbride 🙇

**Review by pgrzesik — APPROVED** [review]
Thank you sir 🙇

**Comment by jameskbride** [comment]
No problem, glad to help! 😃

**Comment by jameskbride** [comment]
@pgrzesik Any chance of getting a release out with this fix in the near future? Thanks!

**Comment by pgrzesik** [comment]
Hey @jameskbride - yes, I intend to prepare a new release later this week, sorry about the delay

**Comment by jameskbride** [comment]
No worries, thanks!

**Comment by william-valencia** [comment]
Any update on the release?

**Pull request merged**